src/commonMethods/constants.js

```diff
@@ -41,4 +41,5 @@ export default {
   suspend: 'suspended',
   WPEFramework: 'WPEFramework',
   blankUrl: 'about:blank',
+  youtubeUrl: 'https://www.youtube.com/tv?v=kg3ElG-H7Wo&resume',
```

Comment:
Is this a particularly long movie? Might rename the var to youtubeLongMovieUrl or something more descriptive.